.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -29,3 +29,11 @@ body:
       description: Mention the environment details (OS, Database, Userstore, etc..) that the product is running on.
     validations:
       required: false
+  - type: checkboxes
+    id: developer-checklist
+    attributes:
+      label: Developer Checklist
+      description: Please confirm the following when closing the issue.
+      options:
+        - label: Will this change make a behavioural change to the product?
```

Comment:
If behavioural change is introduced to the product , let's get the approval from the team lead.

Reply:
Addreessed with: https://github.com/wso2/product-is/commit/a792874bb68ceef6fb5f3dcc138f663993512b38